### PR TITLE
Feature/33 config module

### PR DESCRIPTION
**Pull request by rati3l**
Config module, k8s deployment, and Dockerfile for the app.
Docker image is stored on mu dockerhub for now (ratiel/kdd). To test the k8s deployments it should be enough to run kubectl apply on the manifest file.

I've wrote a short doc in the readme about the variables available.

### CODE REVIEW

**Review by perber — COMMENTED** [review]
Nodeport ist good for the beginning. We should also add later an Ingress configuration. 

We just need to create a ticket for Ingress and Helm.

**Comment by rati3l** [comment]
@perber this is ready for merge

**Review by perber — CHANGES_REQUESTED** [review]

File: internal/config/app_config.go [hunk 1]
```diff
@@ -1,22 +1,88 @@
...
+		if err != nil {
+			zap.L().Fatal("config: unable to bind env: ", zap.Error(err))
+
+		}
 	}
 
-	return &cfg
+	err = viper.Unmarshal(&config)
+	zap.L().Sugar().Infof("Config:\n%+v\n", config)
+	return
```
[the remark this answers is not in the document]
Comment (rati3l) [reply internal/config/app_config.go:87]: I wanted to print config struct, and you need to use sugar for that - data for the regular option needs to be structured.

File: internal/config/app_config.go [hunk 2]
```diff
@@ -1,22 +1,88 @@
...
+	err = viper.ReadInConfig()
 	if err != nil {
-		return nil
+		return
+	}
+
+	// from https://github.com/spf13/viper/issues/188
+	// viper.Unmarshal does not yet handle env var substitution
+
+	env_prefix := "KDD_"
```
[the remark this answers is not in the document]
Comment (rati3l) [reply internal/config/app_config.go]: habits :D i've fixed that

**Review by perber — APPROVED** [review]
lgtm

**Pull request merged**